app/controllers/spotlight/appearances_controller.rb

```diff
@@ -23,19 +23,20 @@ def edit
     protected
 
     def exhibit_params
-      params.require(:exhibit).permit(main_navigations_attributes: [:id, :display, :label, :weight],
```

Comment:
Can we move the `main_navigations_attributes` to the top of the list to avoid making this unnecessary change?